apps/frontend/src/utils/recaptcha/verify-recaptcha.ts

```diff
@@ -10,15 +10,19 @@ const env = envUtil.getEnv();
 export async function verifyRecaptcha(action: string) {
   await ensureGoogleRecaptchaLoaded();
 
-  return new Promise<number>((res, rej) => {
+  if (!env.reCaptcha.siteKey) {
+    return Promise.resolve({ hasValidRecaptchaScore: true });
```

Comment:
Won't it work without Promise.resolve?
`return  {hasValidRecaptchaScore: true }`